README.md

```diff
@@ -1,14 +1,58 @@
 # rivet-add-on-boilerplate
-A starting point for developing Rivet add-ons
+A boilerplate for Rivet add-ons. Replace the contents of this README with information about your add-on, following the template in README.template.md
 
 [Download](https://github.com/indiana-university/rivet-add-on-boilerplate/archive/master.zip) | [Demo](https://indiana-university.github.io/rivet-add-on-boilerplate/)
 
 ## Getting started
-1. Clone or download this repo
-2. Run `npm install`
-3. Run `npm run start` to start devlopment server and watch for changes
+This repo is meant to be cloned, and used as a starting point for a Rivet add-on repo. It contains the javascript and css for the add-on, as well as an example of the HTML required by the component. There are a few steps to get this repo ready to be used as a base.
```

Comment:
- Comma splice: "...meant to be cloned and used as a starting point..."
- Consider "you" instead of "a" ie "...starting point for your Rivet add-on repo..."
- stylize javascript as "JavaScript"
- stylize css as "CSS"
- "HTML required by the component" seems awkward, consider rephrasing to something like "...as well as example HTML snippets to help users start using your component."
- "Get this repo ready to be used as a base" is awkward, consider a new paragraph break/heading and rephrase to something like "Setting up the Rivet add-on boilerplate repo is a four-step process:"